README-dnszone.md

```diff
@@ -161,6 +161,7 @@ ipadnszone
 
 Variable | Description | Required
 -------- | ----------- | --------
+`name` \| `zone_name` | The zone name string. | yes
```

Comment:
Only to keep it more consistent with other modules, move it to line 166, after `ipaadmin_*` itens. Also, add this change to the plugins/ipadnszone.py file (under DOCUMENTATION).

Reply:
Fixed.